tests/flux-v2-image-automation.sh

```diff
@@ -46,6 +46,7 @@ for FILE_LOCATION in $(echo ${FILE_LOCATIONS}); do
     done
 
     ./kustomize build --load_restrictor none "clusters/ptl-intsvc/base" | yq eval 'select(.metadata and .kind == "ImagePolicy")' -  > imagepolicies_list.yaml
+    [ $? -eq 0 ] || exit 1
```

Comment:
Could we have an echo in here prior to exit? Such as:

`echo "kustomize build has failed" && exit 1`